src/rpdk/core/contract/resource_client.py

```diff
@@ -632,3 +634,12 @@ def call(self, action, current_model, previous_model=None, **kwargs):
 
     def has_update_handler(self):
         return "update" in self._schema["handlers"]
+
+    def has_required_handlers_exist(self):
```

Comment:
Can we rename this to "has_required_handlers"? exist seems to be redundant

Reply:
makes sense.